authors/apps/articles/views/ratings_view.py

```diff
@@ -51,7 +51,7 @@ def post(self, request, slug):
             )
         serializer = RatingSerializer(data=rating_field)
         serializer.is_valid(raise_exception=True)
-        if user.id == article_to_be_rated.author.user_id:
+        if user.profile.id == article_to_be_rated.author.user_id:
```

Comment:
I think this line  `if user.profile.id == article_to_be_rated.author.user_id:` is wrong because it compares the user's **profileID** to the **userID** and yet it is a test for ownership of the article. The user's **profileID** and **UserID** are two different things. The test should be `if user.id == article_to_be_rated.author.user_id:`, as you had previously done.

Reply:
this will mean the same since the profile is only created for a new user and they're actually at per with each other so they should possess the same numbering